src/AspNetCoreModuleV2/AspNetCore/src/proxymodule.cxx

```diff
@@ -12,10 +12,6 @@ ASPNET_CORE_PROXY_MODULE_FACTORY::GetHttpModule(
 )
 {
     ASPNET_CORE_PROXY_MODULE *pModule = new (pAllocator) ASPNET_CORE_PROXY_MODULE();
```

Comment:
`new (pAllocator) ` can return null too.

Reply:
good catch.

Reply:
@pan-wang why was pAllocator added to begin with?

Reply:
It's part of IIS interface:
```
HttpModuleFactory
{
public:
    virtual
    HRESULT
    GetHttpModule(
        _Outptr_ CHttpModule **  ppModule,
        _In_ IModuleAllocator *     pAllocator
    ) = 0;

    virtual
    VOID
    Terminate(
        VOID
    ) = 0;
};
```

Reply:
This is IIS style of reusing objects instead of allocating a new one every time.  It is assumed for better performance but not sure whether it is still hold with new compiler and OS